samples/dpdk_app/sriov/README.md

```diff
@@ -0,0 +1,161 @@
+#  SR-IOV Deployment with dpdk-app-centos image
+This directory contains the YAML files needed to start the SR-IOV
+Device Plugin as a daemonset and launch a DPDK based docker image
+which leverages the SR-IOV CNI to plug SR-IOV VFs into the pod. This
+file only describes how these files are used. See the following two
+repos for more details on how SR-IOV Device Plugin and CNI work and
+how to build the SR-IOV Device Plugin image (SR-IOV Device Plugin is
+run in a container as a daemonset) and build the SR-IOV CNI:
+* https://github.com/intel/sriov-network-device-plugin
+* https://github.com/intel/sriov-cni
+
+The files to build the `dpdk-app-centos` image are located in this
+same repo. See the following link for how to build the image and
+details regarding what the image is doing:
+* [dpdk-app-centos](../dpdk-app-centos/README.md)
+
+# SR-IOV Setup
+This test setup assumes you are running on baremetal. Kubernetes, Multus
+and SR-IOV CNI are installed. This test setup uses two physical NICs (PFs),
+and a VF from each PF are attached to the pod. It maps traffic from the PF
+to the each VF using VLANs. 
+
+## Download the sample yaml files
+Use the following steps to download the sample YAML files:
```

Comment:
This is very good example using upstream templates for SR-IOV deployment!
I assume later we may want to have a similar example using openshift or a pointer to openshift doc for SR-IOV deployment.